packages/opentelemetry-shim-opentracing/src/shim.ts

```diff
@@ -85,6 +92,13 @@ export class SpanContextShim extends opentracing.SpanContext {
     return this._spanContext;
   }
 
+  /**
+   * Returns the underlying {@link api.CorrelationContext}
+   */
+  getCorrelationContext(): api.CorrelationContext {
```

Comment:
Why do we have this method if it is not in the original OT api?

Reply:
I'm using it in the inject / extract , here in this file, May be I could use directly the attribute?

Reply:
Ah I see now. No this is fine as it is.